examples/status-message/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "status-message",
+  "version": "1.0.0",
+  "description": "Records the status messages of the accounts that call this contract.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "borsh": "^0.4.0",
+    "near-api-js": "near/near-api-js#bace1ee"
```

Comment:
do we want to use a specific version or is there a reason this commit is used?

Reply:
used to need this commit after current release (May 7 0.41.0): 
![image](https://user-images.githubusercontent.com/13259400/122142039-28782e80-ce81-11eb-870e-25447b3fcd5a.png)

because near-api-js didn't handle `{"result":null}` jsonrpc response correctly. I think I don't need this commit now because I also changed in nearcore side to not return empty response

Reply:
Changed to 0.41.0

Reply:
@austinabell Hmm actually nearcore still returns null, because serde serialize this to null:
```
struct Foo;
```
It serialize `struct Bar {}` to `{}`, so i made this PR: https://github.com/near/nearcore/pull/4387, but before that or near-api-js have a new release, I need to pin near-api-js with this commit.